xonsh/main.py

```diff
@@ -113,7 +113,7 @@ def path_argument(s):
                     metavar='args',
                     help='Additional arguments to the script specified '
                          'by script-file',
-                    nargs='*',
+                    nargs=argparse.REMAINDER,
```

Comment:
ooh this clarifies things, ty! one stackoverflow post is wrong about this and i got confused, it just says:
> Interactive: As the term implies: Interactive means that the commands are run with user-interaction from keyboard. E.g. the shell can prompt the user to enter input. 

so the failling test is actually invalid because i was expecting something different